kinder/README.md

```diff
@@ -1,9 +1,6 @@
 # Kinder
 
-kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
-
-All the kind commands will be available in kinder, side by side with additional commands
-designed for helping kubeadm contributors.
+kinder is a tool designed for helping kubeadm contributors
 
 **kinder is a work in progress. Test it! Break it! Send feedback!**
 
```

Comment:
[1]
it feels to me we should have something like this in the main readme too or possibly move portions from the similar sentence in the new doc:

> NOTE: The exported logic in kinder packages can be a subject of change at any point in time, thus using kinder as a library is unsupported. Please use [kind](https://kind.sigs.k8s.io/) instead.